x-pack/plugins/observability_solution/metrics_data_access/server/routes/metrics_explorer/lib/convert_request_to_metrics_api_options.ts

```diff
@@ -40,6 +40,12 @@ export const convertRequestToMetricsAPIOptions = (
     metricsApiOptions.groupBy = isArray(options.groupBy) ? options.groupBy : [options.groupBy];
   }
 
+  if (options.groupInstance) {
+    metricsApiOptions.groupInstance = isArray(options.groupInstance)
+      ? options.groupInstance
+      : [options.groupInstance];
```

Comment:
What do you think of it?
```suggestion
    metricsApiOptions.groupInstance = [options.groupInstance].flat();
```

Reply:
I think I keep it similar to the rest of the code, maybe we can improve it separately for all of them. (Also, if I am not mistaken, we have a copy of this logic, so we can consider changing that one as well.)

![image](https://github.com/elastic/kibana/assets/12370520/ebba2bca-33fc-4709-99b5-876121a3aa85)